data/storage/storage.go

```diff
@@ -25,6 +26,8 @@ type (
 		Files(ctx context.Context, ptn string) ([]string, error)
 		URL(ctx context.Context, filename string) string
 		String(ctx context.Context, filename string) string
+		GenerateUploadSignedURL(ctx context.Context, filename string, expire time.Duration) (string, error)
```

Comment:
I would like to use the shorten-name `PresignedDownloadURL`, `PresignedUploadURL`, something like that.